app/models/reporting/table.rb

```diff
@@ -16,6 +26,17 @@ def rows
 
     private
 
-    attr_reader :table_hash
+    attr_reader :table_hash, :numeric_column_keys
+
+    def numeric_column?(key)
+      numeric_column_keys.include?(key)
+    end
+
+    # All bar the first column are assumed to be numeric
+    def default_numeric_column_keys
```

Comment:
So say we want to have a non-numeric key for "email" or whatever the way to do this would be to pass a set of numeric keys but miss out the one we want to be non-numeric? If I have this wrong then let me know - but if this is the case is it not a bit opaque? Would it not be clearer to just explictly declare column types 🤔

Reply:
I'd look at it the other way round. If you want a numeric key, then specify which columns you want to be numeric.